src/libPMacc/include/memory/buffers/MultiGridBuffer.hpp

```diff
@@ -156,9 +156,9 @@ class MultiGridBuffer
     }
 
     /**
-     * Starts sync data from own device buffer to neigbhor device buffer.
+     * Starts sync data from own device buffer to neighbor device buffer.
      *
-     * Asynchronously starts syncronization data from internal DeviceBuffer using added
+     * Asynchronously starts synchronization data from internal DeviceBuffer using added
```

Comment:
Could you add `of`:
`starts synchronization of data`
or
`starts synchronizing data`